### PR TITLE
doc: fix nits in blog/release/v8.0.0.md

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
Still not sure about [this](https://github.com/nodejs/nodejs.org/pull/1241#issuecomment-305003661).

### CODE REVIEW

**Review by Trott — APPROVED** [review]
LGTM

File: locale/en/blog/release/v8.0.0.md [hunk 1]
```diff
@@ -200,12 +200,12 @@ AssertionError [ERR_ASSERTION]: false == true
 
 Information for static error codes can be queried quickly by referencing the
 Node.js documentation. For instance, the URL to lookup information about the
-`ERR_ASSERTION` error code is
-https://nodejs.org/dist/latest-v7.x/docs/api/errors.html#ERR_ASSERTION.
+`ERR_ARG_NOT_ITERABLE` error code is
```
Comment (jasnell) [inline locale/en/blog/release/v8.0.0.md:203]: I could have sworn that `ERR_ASSERTION` had been added... will have to make sure it is.

The link format does work tho... try

```
https://nodejs.org/dist/latest-v8.x/docs/api/errors.html#ERR_ARG_NOT_ITERABLE
```

Comment (vsemozhetbyt) [reply locale/en/blog/release/v8.0.0.md:203]: Done)

**Review by lpinca — APPROVED** [review]

**Comment by vsemozhetbyt** [comment]
Should this wait 48 hours?

**Comment by lpinca** [comment]
That rule does not apply to the website but let's wait a little more to give other collaborators a chance to review.

**Pull request merged**